.github/workflows/axiom-byob-clickbench.yml

```diff
@@ -0,0 +1,72 @@
+name: Clickbench Axiom DB on BYOB
+
+on:
+  push:
+    branches:
+      - main
+  schedule:
+    - cron: "2 * * * *"
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}
+
+jobs:
+  clickbench:
+    runs-on: ubuntu-latest
+    env:
+      AXIOM_URL: ${{ secrets.AXIOM_URL_BYOB }}
```

Comment:
Can we only make secret what needs to be? i.e. TOKEN

Reply:
We should create this in the axiom repo I think.

Reply:
do you wanna move everything to axiom repo? i can do that

Reply:
Yeah I think that makes sense.

Reply:
I didn't want the main clickbench to run here and the byob to run in axiom repo for consistency. it is either all lives here or all moved there.